src/components/page.tsx

```diff
@@ -31,12 +33,19 @@ type PageWrapperProps = {
 export const PageTop: React.FC<TitleProps> = ({ tx, ...props }) => {
   const status = tx.tx_status;
   const failed = status === 'abort_by_response' || status === 'abort_by_post_condition';
+  const networkMode = useNetworkMode();
+  // for testnet, show after 4 hours. for mainnet, show after 24 hours
+  const longPending =
+    dayjs().diff(dayjs.unix(tx.receipt_time), 'h') > (networkMode === 'testnet' ? 4 : 24);
```

Comment:
Would be nice to use variables for the 4 and 24 values with descriptive names

Reply:
done!